### PR TITLE
Created tests for blockforPlugin.js

### LINKED ISSUE #560: Resolvers: Create tests for blockForPlugin.js
The Talawa-API code base needs to be 100% reliable. This means we need to have 100% test code coverage.

Tests need to be written for file `lib/resolvers/user_mutations/blockForPlugin.js`

- We will need the API to be refactored for all methods, classes and/or functions found in this file for testing to be correctly executed.
- When complete, all all methods, classes and/or functions in the refactored file will need to be tested. These tests must be placed in a
single file with the name `tests/resolvers/user_mutations/blockForPlugin.spec.js`. You may need to create the appropriate directory structure to do this.

### IMPORTANT:
Please refer to the parent issue on how to implement these tests correctly:

- https://github.com/PalisadoesFoundation/talawa-api/issues/490

### PR Acceptance Criteria

- When complete this file must show **100%** coverage when merged into the code base. This will be clearly visible when you submit your PR.
- [The current code coverage for the file can be found here](https://codecov.io/gh/PalisadoesFoundation/talawa-api/tree/develop/lib/resolvers/user_mutations/). If the file isn't found in this directory, or there is a 404 error, then tests have not been created.
- The PR will show a report for the code coverage for the file you have added. You can use that as a guide.

### PR DESCRIPTION
**Pull request by AbhishekM2001**
**What kind of change does this PR introduce?**

Testing

**Issue Number:**

Fixes #560

**Did you add tests for your changes?**
Yes

**Does this PR introduce a breaking change?**

No

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-api/blob/master/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

**Comment by palisadoes** [comment]
Please fix the failing tests

**Comment by AbhishekM2001** [comment]
Yes, I'm working on it.

**Review by JamaicanFriedChicken — CHANGES_REQUESTED** [review]
tests are still failing, please resolve them~

File: lib/helper_functions/userExists.js [hunk 1]
```diff
@@ -6,7 +6,10 @@ module.exports = async (id) => {
   const user = await User.findOne({ _id: id });
   if (!user) {
     throw new NotFoundError(
-      requestContext.translate('user.notFound'),
+      // requestContext.translate('user.notFound'),
```
Comment (JamaicanFriedChicken) [inline lib/helper_functions/userExists.js:9]: remove commented code

File: lib/resolvers/functions/superAdminCheck.js [hunk 2]
```diff
@@ -5,7 +5,10 @@ const superAdminCheck = (context, user) => {
   const isSuperAdmin = user.userType === 'SUPERADMIN';
   if (!isSuperAdmin) {
     throw new UnauthorizedError(
-      requestContext.translate('user.notAuthorized'),
+      // requestContext.translate('user.notAuthorized'),
```
Comment (JamaicanFriedChicken) [inline lib/resolvers/functions/superAdminCheck.js:8]: remove commented code here

File: lib/resolvers/user_mutations/blockForPlugin.js [hunk 3]
```diff
@@ -1,14 +1,23 @@
...
+const app = express();
+
+
+const blockforPlugin = async (parent, args, context) => {
+  app.use(i18n.init);
   let userFound = await userExists(args.userId);
   if (!userFound) {
     throw new NotFoundError(
-      requestContext.translate('user.notFound'),
+      // requestContext.translate('user.notFound'),
```
Comment (JamaicanFriedChicken) [inline lib/resolvers/user_mutations/blockForPlugin.js:17]: likewise, similar to the above, remove the commented code

File: lib/resolvers/user_mutations/blockForPlugin.js [hunk 4]
```diff
@@ -20,7 +29,10 @@ module.exports = async (parent, args, context) => {
 
   if (!user) {
     throw new NotFoundError(
-      requestContext.translate('user.notFound'),
+      // requestContext.translate('user.notFound'),
```
Comment (JamaicanFriedChicken) [inline lib/resolvers/user_mutations/blockForPlugin.js:32]: remove commented code here as well

File: lib/helper_functions/userExists.js (hunk 1 above)
Comment (AbhishekM2001) [reply lib/helper_functions/userExists.js:9]: ok

File: lib/resolvers/functions/superAdminCheck.js (hunk 2 above)
Comment (AbhishekM2001) [reply lib/resolvers/functions/superAdminCheck.js:8]: ok

File: lib/resolvers/user_mutations/blockForPlugin.js (hunk 3 above)
Comment (AbhishekM2001) [reply lib/resolvers/user_mutations/blockForPlugin.js:17]: ok

File: lib/resolvers/user_mutations/blockForPlugin.js (hunk 4 above)
Comment (AbhishekM2001) [reply lib/resolvers/user_mutations/blockForPlugin.js:32]: ok

**Comment by AbhishekM2001** [comment]
image
I'm facing this error. 
The code is throwing the error as expected from the code but this is failing the test of blockForPlugin.js file.

**Comment by palisadoes** [comment]
Please fix the conflicting file

**Comment by JamaicanFriedChicken** [comment]
Testing is failing, can you resolve it? @AbhishekM2001

**Comment by AbhishekM2001** [comment]
@JamaicanFriedChicken I'm stuck on a bug as you can see in the pic attached in my previous message. It would be a big help if any mentor can look into my code and help me resolve this. Sorry for the delay in response, I've been busy lately.

**Pull request closed without merging**